libgalois/src/PropertyGraph.cpp

```diff
@@ -333,8 +333,11 @@ katana::PropertyGraph::Make(
     return topo_result.error();
   }
 
-  return std::make_unique<PropertyGraph>(
+  auto property_graph = std::make_unique<PropertyGraph>(
       std::move(rdg_file), std::move(rdg), std::move(topo_result.value()));
+  property_graph->recreate_node_property_indexes();
+  property_graph->recreate_edge_property_indexes();
+  return property_graph;
```

Comment:
This is throwing a build error saying unable to convert to  katana result < unique ptr < blah blah
Though the old line 336 was fine
How do i fix this?
@tylershunt

Reply:
Tried a explicit cast in the next push, failed during runtime